### PR TITLE
TSV-226 - Make optional `templateParameters` in `POST /verification-requests` endpoint request on email-verification

### PR DESCRIPTION
**Pull request by anshulbajpai**

### CODE REVIEW

File: it/uk/gov/hmrc/PayloadErrorHandlingISpec.scala [hunk 1]
```diff
@@ -7,6 +7,23 @@ import support.IntegrationBaseSpec
 
 class PayloadErrorHandlingISpec extends IntegrationBaseSpec with GivenWhenThen {
   "request POST on an endpoint" should {
+
+    "not return a BAD request when template parameters are not provided" in {
+      When("a client submits a request without template parameters")
+      stubSendEmailRequest(202)
+      val invalidPayloadWithMissingEmailField = Json.parse(
```
Comment (tamaslang) [inline it/uk/gov/hmrc/PayloadErrorHandlingISpec.scala]: can you remove the val according to the testcase? 
Or any non specific name would be good for me.

Comment (anshulbajpai) [reply it/uk/gov/hmrc/PayloadErrorHandlingISpec.scala]: yes

**Review by paoloboni — APPROVED** [review]
I'm ok with these changes

**Review by scottkwalker — APPROVED** [review]
👍

**Pull request merged**